babelon-convert.py

```diff
@@ -0,0 +1,21 @@
+import click
+
+
+@click.command()
+@click.option("--input", "-i", "in_file", required=True,
+              help="Path to csv file to be processed.",
+              type=click.Path(exists=True, dir_okay=False, readable=True))
+@click.option("--output-file", "-o", default="./output.xlsx",
```

Comment:
default should be tsv (in particular, babelon conformant tsv), but no worries now :)

Reply:
I should have been more clear on the pull request comment and not include files that aren't finished, this PR is only for updating the xml parser code - notebook (I have not updated the CLI code yet). I will update this PR.